docs/macbuild.rst

```diff
@@ -14,22 +14,25 @@ distribution to other systems.
    ``Allow apps downloaded from Anywhere``. This is necessary because
    GateKeeper will try to prevent this installation. You can return this
    setting to its previous state following the installation.
```

Comment:
Are the Python installers still causing issues with GateKeeper? I would be surprised if this hadn't been fixed in which case we could remove this step.